watchtower-plugin/src/main.rs

```diff
@@ -379,8 +387,10 @@ async fn on_commitment_revocation(
                             let mut state = plugin.state().lock().unwrap();
                             state.set_tower_status(tower_id, TowerStatus::TemporaryUnreachable);
                             state.add_pending_appointment(tower_id, &appointment);
-
-                            state.unreachable_towers.send(tower_id).unwrap();
+                            state
```

Comment:
But won't the `TemporaryUnreachable` towers miss any new appointments we send later?
So the race condition is still there?

Reply:
You actually reviewed an old version of this where `unreachable_towers` was still holding only `tower_id` instead of `(tower_id, locator)`. My bad for not flagging this as a draft.